package.json

```diff
@@ -3,41 +3,48 @@
   "version": "1.0.0",
   "description": "Simple inventory management system for any business",
   "scripts": {
-    "start": "node ./scripts/www",
+    "start": "node scripts/server.js",
     "build": "react-scripts build",
-    "dev-fe": "react-scripts start",
-    "dev-be": "nodemon ./scripts/www",
+    "dev-fe": "NODE_ENV=development,EXTEND_ESLINT=true react-scripts start",
+    "dev-be": "NODE_ENV=development,EXTEND_ESLINT=true nodemon scripts/server.js",
     "test": "react-scripts test",
     "eject": "react-scripts eject",
-    "postinstall": "npm run build"
+    "postinstall": "npm run build",
+    "lint": "eslint --ext .js,.jsx .",
+    "lint-fix": "eslint --ext .js,.jsx --fix ."
   },
   "dependencies": {
     "axios": "^0.19.2",
+    "bcrypt": "^5.0.0",
+    "body-parser": "^1.19.0",
     "bootstrap": "^4.5.0",
     "cookie-parser": "~1.4.4",
     "debug": "~2.6.9",
     "dotenv": "^8.2.0",
     "express": "~4.16.1",
     "express-session": "^1.17.1",
     "http-errors": "~1.6.3",
-    "jade": "~1.11.0",
-    "lodash": "^4.17.15",
+    "jsonwebtoken": "^8.5.1",
+    "lodash": "^4.17.19",
+    "moment": "^2.27.0",
     "morgan": "~1.9.1",
     "mysql": "^2.18.1",
     "passport": "^0.4.1",
+    "passport-jwt": "^4.0.0",
     "passport-local": "^1.0.0",
     "react": "^16.13.1",
     "react-bootstrap": "^1.1.1",
     "react-dom": "^16.13.1",
     "react-router-dom": "^5.2.0",
-    "react-scripts": "3.4.1"
+    "react-scripts": "3.4.1",
+    "yup": "^0.29.1"
   },
   "devDependencies": {
     "@testing-library/jest-dom": "^4.2.4",
     "@testing-library/react": "^9.3.2",
     "@testing-library/user-event": "^7.1.2",
     "acorn": "^7.3.1",
-    "babel-eslint": "^10.1.0",
```

Comment:
I thought we were going to add babel-eslint?

Reply:
This was before our discussion

Reply:
okay was not sure.

Reply:
Nah, it's a good callout, will re-add